daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -111,7 +129,10 @@ private[lf] object PartialTransaction {
       nodeId: NodeId,
       parent: Context,
       byKey: Boolean,
-  )
+  ) extends ContextInfo {
+    override private[PartialTransaction] def childrenSeeds(idx: NodeIdx) =
+      crypto.Hash.deriveNodeSeed(parent.nextChildrenSeed, idx)
```

Comment:
hm, I would find it more intuitive if we crash when constructing `ExercisesContextInfo` instead of only when we call `childrenSeeds`. Is there an issue with doing that?

Reply:
Good point.

I changed the code, so we use `nextChildrenSeed` only once and crash when constructing.